rtf_vis_tool/README.md

```diff
@@ -29,7 +35,7 @@ Note: running the application for the first time will auto generate a `/node_mod
 - `node_modules`: internal packages used throughout the application. Don't edit these.
 - `public`: contains index.html file 
 - `src`
-    - `Components`: contains all React components used for rendering graph nodes, summaries, point cloud viewers, etc...
+    - `Components`: contains all React components used for rendering graph nodes, summaries, point cloud viewers, etc. Each file within this folder returns a format JSX, which is essentially a modified version of HTML that is meant to be readable by React code
```

Comment:
emphasize that index.html contains the initial view that will be displayed, and that it will start App.js, which is the top-level component. Only keep App.js and rename it to `GtsfmVisualizer.js`

Reply:
To make things simpler, I've just removed `App.js` all together (since it was never really necessary). Now the hierarchy is 
`index.html` -> `index.js` -> `LandingPageGraph.js`